gatsby-browser.js

```diff
@@ -5,3 +5,21 @@
  */
 
 // You can delete this file if you're not using it
+
+/**
+ * Implement Gatsby's SSR (Server Side Rendering) APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/ssr-apis/
+ */
+
+import React from "react"
+import {Theme} from './src/components/Theme'
+
+
+export const wrapRootElement = ({ element }) => {
+  return (
+    <Theme>
+      {element}
+    </Theme>
```

Comment:
This should go in both gatsby-browser.js and gatsby-ssr.js. I'm not totally clear how these two points differ, sadly... As you can see here i copy-pasted the comment as well. Oops.